sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -24,7 +24,7 @@
 
 n_clusters = 3
 centers = np.array([[1, 1], [-1, -1], [1, -1]]) + 10
-X, _ = make_blobs(n_samples=300, n_features=2, centers=centers,
+X, _ = make_blobs(n_samples=50, n_features=2, centers=centers,
                   cluster_std=0.4, shuffle=True, random_state=11)
```

Comment:
Can't we just re-run make_blobs with these parameters in `test_parallel`? Otherwise, this would change the `X` for all `mean_shift` tests, and I'm not familiar enough with though to say whether this could have unintended side effects..